src/Pages/Home/index.tsx

```diff
@@ -106,7 +108,7 @@ export default function Home() {
         window.open(whatsappUrl, "_blank");
     };
 
-    if(isLoading) return <LoadingFullScreenTemplate />
+    if (isLoading) return <LoadingFullScreenTemplate />
 
     if (scheduleInformation.isServiceOpen) {
         return (
```

Comment:
O que acha de fazer assim?

```suggestion
    if (scheduleInformation.isServiceOpen) {
        const PARKING_CAPTION_COMPONENT = <Caption isLink icon={<FontAwesomeIcon color='#0088c2' icon={faCircleInfo} />} label='Como encontrar minha vaga?' />
        return (
```

Reply:
Continua embaixo...